community/sigla/about.en.md

```diff
@@ -10,14 +10,18 @@ old_url: http://www.rism.info/en/sigla/about-sigla.html
 
 Since 1960, RISM has published catalogs with descriptions of musical source material from libraries and archives in many countries around the world. From the very beginning, institutions that hold musical sources were given an abbreviation, called a library siglum. The RISM library sigla make it possible to concisely name holding institutions – sometimes a large number of them – without having to cite very long institutional names. We do not require sources to be in the RISM catalog in order for a siglum to be created, and the materials do not necessarily have to be old.
 
-A siglum consists of uppercase letters, which designate the country and city, and lowercase letters, which stand for the institution. The code for the country is taken from the list of international vehicle registration codes maintained by the United Nations. City and institution codes are created by the RISM Editorial Center.
+RISM sigla are composed of a country abbreviation, a city abbreviation, and (usually) an institution abbreviation. The country abbreviations are taken from the [Vehicle Registration Identification code](https://en.wikipedia.org/wiki/International_vehicle_registration_code). The city identifier is assigned by RISM and is unique to a country. It may be helpful to look at a few examples.
 
-For example:\
-F-Pn = **F**rance, **P**aris, Bibliothèque **n**ationale de France, Département de la Musique.
+In **I-Rc**, **I** stands for Italy, **R** is for Rome, and **c** is for the [Biblioteca Casanatense](https://rism.online/institutions/30000230).
 
-In the case of private collections, the entire last name of the owner is added to the city code:  
-I-PEbattisti = **I**taly, **Pe**rugia, Biblioteca privata Renzo **Battisti**.
+In **US-PHf**, **US** stands for the United States, **PH** is for Philadelphia, and **f** is for the [Parkway Central Library, Music Department](https://rism.online/institutions/30002414).
```

Comment:
I think I would leave the `f` example here, since (in theory) the institution identifier doesn't **have** to match with the name... it just has to be unique. It's better to reinforce that it's just a letter, and users don't have to try and guess what combination of letters were used. 

(It's also the original example, but corrected -- and it is still part of the Free Library...)

Reply:
I think it would make sense to use another example since we have thousands where the letters somehow make sense. We could use that one as one as an additional one where it does not to explain that it does not always make sense - for various reasons.

Reply:
My intention with `US-PHf` was just to show an example from the US and how even here the city is more prominent than a state (which might be the expectation for many US users). I didn't catch that the name changed. If I were rewriting the help text I'd just pick a different example, like `US-PHu` maybe. https://rism.online/institutions/30002433
University of Pennsylvania Libraries